infra/build_specified_commit.py

```diff
@@ -152,6 +157,33 @@ def _build_image_with_retries(project_name):
   return result
 
 
+def get_required_post_checkout_steps(dockerfile_path):
+  """Get required post checkout steps (best effort)."""
+
+  checkout_pattern = re.compile(r'\s*RUN\s*(git|svn|hg)')
+
+  # If the build.sh is copied from upstream, we need to copy it again after
+  # changing the revision to ensure correct building.
+  post_run_pattern = re.compile(r'\s*RUN\s*(.*build\.sh.*(\$SRC| /src).*)')
```

Comment:
space between | and /src, is this intended ?

Reply:
Removed, this was needed in a previous iteration but no longer required.